Projects/Windchill-App.md

```diff
@@ -0,0 +1,29 @@
+# Windchill
+
+Windchill combines the actual temperature with the wind speed to calculate
+the windchill factor, or what the perceived temperature is versus the actual
+temperature.
+
+## User Stories
+
+-   [ ] User can enter the actual temperature (Celcius) and the wind speed
+(kilometers per hour)
+-   [ ] User can press the `Calculate` button to display the wind chill
+-   [ ] User will receive and error message if data values are not entered
+or if the resulting wind chill factor is greater than or equal to the
+actual temperature.
+-   [ ] User will be prompted to enter new data values if `Calculate` is 
```

Comment:
Maybe move this in to the bonus features and the `English metric` to be a User story as it seems much more of a "must" to have metric system also... What do you say? 🤔

Reply:
Agree.

Reply:
Great!